examples/resx/html/index.html

```diff
@@ -0,0 +1,35 @@
+<html lang="zh-cn" window-resizable="false" window-icon="resx://html/icon/i32.png">
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+    <link rel="stylesheet" href="resx://html/css/main.css">
+    <script src="resx://html/js/zepto.js"></script>
+    <script src="resx://html/js/util.js"></script>
+</head>
+<body>
+<div class="main">
+    <div>
+        <button id="btn1">按钮1</button>
+        <button id="btn2">按钮2</button>
+    </div>
+    <div style="margin-top: 30px;">
+        <textarea id="out"></textarea>
+    </div>
+</div>
+</body>
+
+<script>
+    util.moveCenter(800, 500);
+    util.ready(function (view) {
+        $('#btn1').on('click', function () {
+            $('#out').val("按钮1点击")
+        });
+        $('#btn2').on('click', function () {
+            $('#out').val("按钮2点击")
+        });
+    })
+</script>
+
+</html>
```

Comment:
If you really want to add this entire example to this repository, would you mind creating an English version instead? Also I think this `resx` example can be added on a separate PR not this where we are trying to resolve the building issues